dpp/src/state_repository.rs

```diff
@@ -59,7 +59,7 @@ pub trait StateRepositoryLike: Send + Sync {
     /// Remove Document
     async fn remove_document(
         &self,
-        contract_id: &Identifier,
+        contract_id: &DataContract,
```

Comment:
Why the name is `contract_id`, but the type is `DataContract`? Maybe someone who worked on the original implementation can clarify this? @shumkov @jawid-h @shuplenkov

Reply:
Alright, I had a chat with @shumkov, and it looks like it needs to be renamed to `data_contract`

Reply:
thanks, I was changing things too fast